hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -1307,3 +1309,49 @@ def hpat_pandas_series_abs_impl(self):
         return pandas.Series(numpy.abs(self._data))
 
     return hpat_pandas_series_abs_impl
+
+
+@overload_method(SeriesType, 'unique')
+def hpat_pandas_series_unique(self):
+    """
+    Pandas Series method :meth:`pandas.Series.unique` implementation.
+
+    Note: Return values order sorted by values
+
+    .. only:: developer
+
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_unique
+             python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_unique_str
+
+    Parameters
+    -----------
+    self: :class:`pandas.Series`
+        input arg
+
+    Returns
+    -------
+    :obj:`numpy.array`
+         returns :obj:`numpy.array` ndarray
+    """
+
+    _func_name = 'Method unique().'
+
+    if not isinstance(self, SeriesType):
+        raise TypingError(
+            '{} The object must be a pandas.series. Given self: {}'.format(_func_name, self))
+
+    if self.dtype is types.unicode_type:
+        def hpat_pandas_series_unique_str_impl(self):
+            unique_strings = init_set_string()
```

Comment:
What type returned by `init_set_string()`?
Is it ok to call `.add` on this type?